### PR TITLE
fix: use consistent title

### LINKED ISSUE #202: Add proper page title
`Create Next App` ...

### PR DESCRIPTION
**Pull request by MaxSchaefer**
Use `helpwave tasks` or the suffix `~ helpwave tasks`

Closes #202

### CODE REVIEW

**Review by florian-sabonchi — COMMENTED** [review]
I think we should vote again whether to take a pipe or a tilde

File: tasks/pages/profile.tsx [hunk 1]
```diff
@@ -54,7 +54,7 @@ const ProfilePage: NextPage = () => {
   return (
     <PageWithHeader>
       <Head>
-        <title>Settings</title>
```
Comment (DasProffi) [inline tasks/pages/profile.tsx:57]: profile should be translated

File: tasks/pages/settings.tsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ const SettingsPage: NextPage = () => {
   return (
     <PageWithHeader>
       <Head>
-        <title>Settings</title>
```
Comment (DasProffi) [inline tasks/pages/settings.tsx:31]: settings needs to be translated

**Review by DasProffi — CHANGES_REQUESTED** [review]

**Comment by DasProffi** [comment]
Rather than hardcoding the suffix on every page we should use a function to wrap the name of the page
```
const titleWrapper = (title: string) => title + '~ helpwave tasks'
```
This should allow us to easily modify it

File: tasks/pages/profile.tsx (hunk 1 above)
Comment (use-to) [reply tasks/pages/profile.tsx:57]: profile page is legacy code #183

**Comment by MaxSchaefer** [comment]
@DasProffi @florian-sabonchi
` ~ ` yes or no? The alternatives are ` | ` or ` - `.

**Review by DasProffi — APPROVED** [review]

**Pull request merged**